META.md

```diff
@@ -67,3 +68,30 @@ Create a file for the issue body, e.g. `issue.md`.
 ```
 meta exec "gh issue create --label backwards-compatibility --title 'Ensure backwards compatibility with ODFE' --body-file ../issue.md"
 ```
+
+### Open a Pull Request in Each Repo
+
+In [opensearch-build#497](https://github.com/opensearch-project/opensearch-build/issues/497) we needed to remove `integtest.sh` from each repo.
+
+We'll be pushing to forks. Make sure to replace `username` with your GitHub username below and to fork all the repos first.
+
+```
+meta exec "gh repo fork"
+```
+
+Ensure that a remote is setup for each plugin pointing to our forks.
+
+```
+meta exec "git remote get-url origin | sed s/opensearch-project/username/g | xargs git remote add username"
+```
+
+Remove a file, e.g. `integtest.sh`, commit and push.
+
+```
+meta exec "git rm integtest.sh"
+meta exec "git add --all"
+meta exec "git checkout -b remove-integtest-sh"
+meta exec "git commit -s -m 'Removed integtest.sh.'"
+meta exec "git push username remove-integtest-sh"
+meta exec "gh pr create --title 'Removing default integtest.sh.' --body='Coming from https://github.com/opensearch-project/opensearch-build/issues/497, removing default integtest.sh.'"
```

Comment:
Looks nice! Curious - If a repo doesn't have the integtest.sh file, is that gracefully handled with this?

Reply:
Yeah, it just fails through. If there's nothing to PR, it doesn't do anything.